### DISCUSSION TITLE
Lists

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#38370] [Other Feature Feedback, Questions, & Ideas]
What are lists?

**Comment by david-engelmann** [comment]
[Lists level up the starring experience by making it easy to organize and curate your favorite repositories on GitHub.](https://github.blog/changelog/2021-12-09-lists-are-now-available-as-a-public-beta/). Check out this blog post on it!

**Comment by jfullstackdev** [comment]
say there is no Lists and you want to star repos that you like

then you realized it's becoming big, it's now a collection but there is no way to organize, they are just random repos on the Stars tab

without Lists, how will you group them ? it can be done in another repo and including the repos of the same topic or as per your category like, My Favorite Projects

but it's not handy, so GitHub offered this Lists so it's handy now
